google-account-plugin/resources/META-INF/plugin.xml

```diff
@@ -20,8 +20,8 @@
   <!-- "version" set by gradle-intellij-plugin -->
   <!-- "idea-version since-build" set by gradle-intellij-plugin -->
   <!-- Workaround for gradle-intellij-plugin limitation: https://github.com/JetBrains/gradle-intellij-plugin/issues/66 -->
-  <!-- Setting version to 15.0.3 and up. https://confluence.jetbrains.com/display/IDEADEV/IntelliJ+IDEA+15+143.1821.5+Release+Notes -->
-  <idea-version since-build="143.1821.5" />
+  <!-- Setting version to 15.0.6 and up. https://confluence.jetbrains.com/display/IDEADEV/IntelliJ+IDEA+15+143.1821.5+Release+Notes -->
```

Comment:
@patflynn - no, see https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/447

The basic idea is that the plugin doesn't currently provide a way to configure the "since" and "until" portions of the version selection to achieve what we want so we are manually entering the versions for now. Once the plugin is updated (https://github.com/JetBrains/gradle-intellij-plugin/issues/66#issuecomment-214879784).